cadence/tests/tchannel_to_grpc_migration/list_domains.py

```diff
@@ -8,14 +8,9 @@
         next_page_token=bytes(0)
     )
     wfc = WorkflowService.create('localhost', 7833, 300, PROTOCOL_GRPC)
-    result = wfc.list_domains(request)
-    for domain in result[0].domains:
-        print(domain.domain_info.uuid)
-        print(domain.domain_info.name)
-
-
+    grpc_result, errorOne = wfc.list_domains(request)
     wfc = WorkflowService.create('localhost', 7933, 300, PROTOCOL_TCHANNEL)
-    result = wfc.list_domains(request)
-    for domain in result[0].domains:
-        print(domain.domain_info.uuid)
-        print(domain.domain_info.name)
\ No newline at end of file
+    tchannel_result, errorTwo = wfc.list_domains(request)
```

Comment:
should we also assert errorOne and errorTwo being None?

Reply:
Good idea. I'll do it in a follow up PR after [this one](https://github.com/cadence-oss/cadence-python/pull/7/files#diff-5f25f1d379760b87b6fc57e39e9592ed033edba494f7fedcf9faa225b0305039R46), I added a todo note [here](https://github.com/cadence-oss/cadence-python/pull/7/files#diff-5f25f1d379760b87b6fc57e39e9592ed033edba494f7fedcf9faa225b0305039R46).  Next diff will be do fix small changes like that.